tests/subsys/dsp/basicmath/testcase.yaml

```
@@ -1,6 +1,6 @@
 tests:
   libraries.cmsis_dsp.basicmath:
```

These are no longer CMSIS-DSP tests, so the test names need to be updated as well:

```suggestion
  dsp.basicmath:
```

---

Done